classes/form/CustomerForm.php

```diff
@@ -73,6 +77,17 @@ public function setPasswordRequired($passwordRequired)
         return $this;
     }
 
+    public function fillWith(array $params = [])
+    {
+        if (isset($params['email'])) {
```

Comment:
```suggestion
        if (!empty($params['email'])) {
```
We don't care about 0, false and empty string :)